build_apache_pyavro.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+
+# Copyright 2015 Byhiras (Europe) Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE(Christian):
+
+# The following are the commits of Tubular/avro that are *different* from apache/avro,
```

Comment:
Why not just rebase/merge origin/master into our master? I understand the point of bazel: to run on last version of things, but that's about our own packages, we still freeze external dependencies right? I just mean won't this give us more problems than benefits?

Reply:
this gives us both problems AND benefits but not sure which one weighs heavier -- I tried a rebase initially but it ends up being a royal pain to go through the various breaking diffs -- I spent about 20/30 minutes and then gave up. I'd rather not touch the code at all but this is one way to do it in which we can easily drop a patch as it gets merged in...